tools/online_editor/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "syntax_check": "tsc --noemit",
```

Comment:
can we run it in CI?
alternatively , can we configure typescript compiler to actually error out in the CI?

Reply:
actually, it seems another commit do that, so that should be fine

Reply:
No, it now the Ci errors out because I somehow broke the editor worker for Monaco.

Reply:
added that and the linter run to the CI in the meantime.